wpilib/wpilib/digitalinput.py

```diff
@@ -32,19 +33,23 @@ def __init__(self, channel):
         """
         # Store the channel and generate the handle in the constructor of the parent class
         # This is different from the Java implementation
-        super().__init__(channel, True)
+        super().__init__()
```

Comment:
You realise the comment no longer matches, right?

Reply:
Nope